assets/javascripts/test_result.js

```diff
@@ -729,6 +729,33 @@ function renderCommentsTab(response) {
   const tabPanelElement = this.panelElement;
   tabPanelElement.innerHTML = response;
   $(tabPanelElement).find('[data-toggle="popover"]').popover({html: true});
+  // Add job status icons to /t123 urls
+  const hostname = $(location).attr('host');
+  $(tabPanelElement)
+    .find('a')
+    .each(function (index, element) {
+      const href = $(element).attr('href');
+      if (href === undefined) {
+        return;
+      }
+      const re = new RegExp('^https?://' + hostname + '/t([0-9]+)$');
+      const found = href.match(re);
+      if (!found) {
+        return;
+      }
+      const id = found[1];
+      const url = '/api/v1/jobs/' + id + '/status';
+      $.ajax(url)
```

Comment:
I suppose it would make sense if there was just one request (for all IDs), e.g. we could allow a list of comma-separated IDs here and return the results in an array.

Reply:
True. The use case this is for will usually have 1-4 such links. We might improve this later and can monitor the Apache response times for now.

Reply:
Ok. We could allow calling it like `/api/v1/jobs/1,2,3/status` in the future which would still be backwards compatible. However, the response format should better be already changes so we don't have to distinguish the one ID and multiple IDs cases later. Or you just move the route under some experimental prefix for now to we can break compatibility later (e.g. we have already `/api/v1/experimental/search`).

Reply:
Hmm, I'm not really a fan of `/api/v1/jobs/1,2,3/status`.
I vote for going experimental.

Reply:
done